scripts/setup

```diff
@@ -8,19 +8,17 @@ fi
 
 function usage() {
     echo -n \
-         "Usage: $(basename "$0")
+"Usage: $(basename "$0")
+
 Attempts to setup the project's development environment.
 "
 }
 
-if [ "${BASH_SOURCE[0]}" = "${0}" ]
-then
-    if [ "${1:-}" = "--help" ]
-    then
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
         usage
     else
-        # Ensure that a directory exists to house AWS profiles.
-        mkdir -p "${HOME}/.aws"
+        # https://github.com/raster-foundry/raster-foundry/pull/3800
```

Comment:
I would leave an inline comment here that describes the gist of that PR vs. linking to it. You should also do both, but at least having a brief description helps those who just want to skim.